govuk-one-login/tech-docs
# AP-408 - remove sample reference material page

[AP-408](https://govukverify.atlassian.net/browse/AP-408)

## Why

The guidance on the reference material page is not helpful to service teams

## What

remove https://docs.sign-in.service.gov.uk/use-sample-reference-material/ https://github.com/govuk-one-login/tech-docs/blob/main/source/use-sample-reference-material.html.md.erb from the tech docs

![image](https://github.com/user-attachments/assets/f2d303cd-53b0-4972-9d7f-f7fe1459c435)


## Technical writer support

Do you need a tech writer's support, for example to review your PR? *YES*

## How to review

- `./preview-with-docker.sh`
- open http://localhost:4567
- confirm that the page has been removed

## Changelog

none required

## Confirm



[AP-408]: https://govukverify.atlassian.net/browse/AP-408?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
